plugins/inputs/webhooks/webhooks.go

```diff
@@ -0,0 +1,81 @@
+package webhooks
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	_ "github.com/influxdata/telegraf/plugins/inputs/webhooks/webhooks_all"
+	"github.com/influxdata/telegraf/plugins/inputs/webhooks/webhooks_models"
+)
+
+func init() {
+	inputs.Add("webhooks", func() telegraf.Input { return NewWebhooks() })
+}
+
+type Webhooks struct {
+	ServiceAddress string
+
+	Webhook []WebhookConfig
+}
+
+type WebhookConfig struct {
```

Comment:
We changes the suggested configuration because we couldn't find a nice and effective way to easily detect configured webhooks plugins with this configuration.

Correct me if i am wrong, but with this configuration i would have a `struct` like this : 

```go
type WebhookConfig struct {
    ServiceAddress string
      
    RollbarConfig RollbarConfig
    GithubConfig  GithubConfig
}

type RollbarConfig struct {
     path string
}

type GithubConfig struct {
     path string
}
```

Even with this configuration:

```go
[[inputs.webhook]]
  service_address = ":1619"
  [inputs.webhooks.github]
    path = "/github"
````

The problem is:
 
1. The `RollbarConfig` get initialized so we don't know if we should start it or not. 
2. Furthermore, this probably mean that we can't iterate over a list of webhooks plugins and we have to manually iterate like this: 

```go
plugins := [wb.GithubConfig, wb.RollbarConfig]
for plugin := range plugins {
    plugin.Register...
}
````

Do you have any ideas of how we could solve this?